salt/modules/boto_elbv2.py

```diff
@@ -76,7 +72,7 @@ def __virtual__():
     """
     has_boto_reqs = salt.utils.versions.check_boto_reqs()
     if has_boto_reqs is True:
-        __utils__["boto3.assign_funcs"](__name__, "elbv2")
+        salt.utils.boto3mod.assign_funcs(__name__, "elbv2")
```

Comment:
Is `__utils__` not loaded in `__virtual__` in all cases? It looks like we do load that before modules.

Reply:
I think it's a race condition...

Reply:
Apparently `__utils__` is only loaded in the `__virtual__` function for state modules... not execution modules.